content/webapp/components/IIIFViewer/MainViewer.tsx

```diff
@@ -468,6 +469,12 @@ const MainViewer: FunctionComponent = () => {
     }
   }, [canvas]);
 
+  useEffect(() => {
+    // Keeping this very simple for now - I think it'll just apply to audio, video and of course mixed works containing either of them.
+    // I would like to also have it return false for PDFs, but they currently do have a thumbnail image.
+    setHasImagesOnly(!canvases?.find(canvas => !canvas.thumbnailImage?.url));
```

Comment:
@gestchild is there a better indicator that doesn't require complicated logic? I started playing with going through all canvases and seeing if they had Original as well as painting etc but it felt like there's so many different types that it might not really work anyway?